### PR TITLE
[PAGOPA-1800] fix: Update Stamp mapping

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- Update and check all responses

#### Motivation and Context

- `Stamp` object mapping fix

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/debtposition/controller/payments/api/impl/PaymentsController.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
     @Override
     public ResponseEntity<PaymentOptionWithDebtorInfoModelResponse> getOrganizationPaymentOptionByNAV(
             String organizationFiscalCode, String nav) {
-        log.info(String.format(LOG_BASE_HEADER_INFO, "GET", "getOrganizationPaymentOptionByIUV", String.format(LOG_BASE_PARAMS_DETAIL, organizationFiscalCode, nav)));
+        log.info(String.format(LOG_BASE_HEADER_INFO, "GET", "getOrganizationPaymentOptionByNAV", String.format(LOG_BASE_PARAMS_DETAIL, organizationFiscalCode, nav)));
```
[the remark this answers is not in the document]
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/debtposition/controller/payments/api/impl/PaymentsController.java:42]: Done via annotation
```
@Pattern(regexp = "\\d{1,30}") String organizationFiscalCode, @Pattern(regexp = "^\\d{1,30}$") String nav
```

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 👍

**Review by pasqualespica — APPROVED** [review]
ok even if the test coverage is passed, we don't see any specific test on "stamp" feature .
Right ?

File: src/main/java/it/gov/pagopa/debtposition/controller/payments/api/IPaymentsController.java [hunk 2]
```diff
@@ -33,8 +34,8 @@ public interface IPaymentsController {
     @GetMapping(value = "/organizations/{organizationfiscalcode}/paymentoptions/{nav}",
             produces = {"application/json"})
     ResponseEntity<PaymentOptionWithDebtorInfoModelResponse> getOrganizationPaymentOptionByNAV(
-            @Parameter(description = "Organization fiscal code, the fiscal code of the Organization.", required = true)
-            @PathVariable("organizationfiscalcode") String organizationFiscalCode,
+            @Pattern(regexp = "\\d{1,30}") @Parameter(description = "Organization fiscal code, the fiscal code of the Organization.", required = true)
```
[the remark this answers is not in the document]
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/debtposition/controller/payments/api/IPaymentsController.java:37]: JUnit tests have been added. The check `\\d{1,30}` was added just for security purposes (log injection)

**Pull request merged**